tools/autograd/derivatives.yaml

```diff
@@ -431,9 +431,11 @@
 
 - name: cummax(Tensor self, int dim) -> (Tensor values, Tensor indices)
```

Comment:
(note to self): here's a list of all the formulas in this diff: https://gist.github.com/zou3519/f5e818f8586854f6d018e52398920fa8